tests/TestingTools/TaskBuilder.ts

```diff
@@ -83,6 +84,7 @@ export class TaskBuilder {
             tags: this._tags,
             originalMarkdown: '',
             scheduledDateIsInferred: this._scheduledDateIsInferred,
+            createdDate: this._createdDate,
```

Comment:
Please update this according to:
https://publish.obsidian.md/tasks-contributing/Code/How+do+I+add+a+new+field+to+the+Task+class

It needs the corresponding method to allow tests to control the new field.

Reply:
* Change identicalTo method
* Add tests in Task.tests.ts